src/constants/RecipeDefaultValues.ts

```diff
@@ -1,19 +1,12 @@
 import { RecipeFormValues } from "../types/FormTypes";
 import { uniqueId } from "../utils/recipeUtils";
 
-function createEmptyFileList(): FileList {
-  const files: File[] = [];
-  return {
-    item: () => null,
-    ...files,
-  };
-}
 
 export const recipeDefaultValues: RecipeFormValues = {
   title: "",
   time: "",
   description: "",
-  image: createEmptyFileList(),
+  imgSrc: import.meta.env.VITE_DEFAULT_DISH_URL,
```

Comment:
By the name i am unsure if this should be kept in env, or just in const 🤔 
look into it, and if there is a reason then Okay ^^